src/components/ImagesSlider/ImageSlider.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Swiper, SwiperSlide } from "swiper/react";
+import styles from "./styles.module.css";
+import "swiper/css";
+import "swiper/css/effect-coverflow";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
```

Comment:
should we import the 3 lines below in seperate lines as we already import this line "import "swiper/css";" ?
import "swiper/css/effect-coverflow";
import "swiper/css/pagination";
import "swiper/css/navigation";

Reply:
yes , because  pagination and navigation seems to be classes from the swiper css . and when i am not importing them a problem with styles will happen and the arrows under the slider will disappear .